handbook/ops/bizops/user_definitions.md

```diff
@@ -1,37 +1,67 @@
 # User metrics definitions
 
-## On-premise
+We use metrics to guide prioritization and planning. By defining metrics against the kinds of behaviors that we want to drive, we can plan features and measure results against the outcomes we're looking for. The three most important metrics we track are acquisition (of visitors), activation (turning them into active users), and retention (keeping them month to month).
 
-### Metrics definitions
+Specific targets for FY22 can be found on the [product home page](../../product/index.md#goals).
 
-**On premise**: a user is considered 'active' when they are signed-in and trigger any event over the specified time period. This could be anything from a page view inside the product to a hover using one of Sourcegraph's browser extensions to a code monitoring email notification.
+## User states
 
-Time periods we track:
-
-- Daily active user (DAU)
-- Weekly active user (WAU)
-- Monthly active user (MAU)
-
-### Growth metrics user categories
-
-We track the following categories in pings for each month. The explanations are framed as though we're talking about the current month, but apply to any given month.
+Users can be in a number of different states. We track the following categories in pings for each month. The descriptions below are framed as though we're talking about MAU, but the same definitions apply to whatever period is being considered.
 
 | Metric      | Description                                                      |
 | ----------- | ---------------------------------------------------------------- |
+| Visitor     | Users who accessed the site                                      |
+| Active      | Users who performed a qualifying event (see below)               |
 | Retained    | Users who were active last month _and_ this month                |
 | Churned     | Users who were active last month but not this month              |
 | Resurrected | Users who were _not_ active last month but are active this month |
 | Created     | Users whose account was created this month                       |
 | Deleted     | Users whose account was deleted this month                       |
 
-### Engagement ratios
+### What is a visitor?
+
+A visitor is a unique person who showed up on the site and did anything (or nothing at all, apart from viewing the page.) This is important to measure because each of these users is potentially an active user, but it's also important that we don't count anyone who lands on the site as an active user so that we can measure success of our features that are intended to convert them.
```

Comment:
In this section to accommodate that we are describing metrics for both on-premises and cloud solutions could we replace the word "site" with "product" or something that does not infer a "location"?  Reach out if this feedback is unclear.

Reply:
```suggestion
A visitor is a unique person who showed up on the site and did anything (or nothing at all, apart from viewing the page.) This is important to measure because each of these users is potentially an active user, but it's also important that we don't count anyone who uses the product as an active user so that we can measure success of our features that are intended to convert them.
```